app/src/main/java/com/aptoide/uploader/apps/InstalledApp.java

```diff
@@ -232,12 +232,7 @@ public void setObbList(List<Obb> obbList) {
     List<FileToUpload> list = new ArrayList<>();
     list.add(new FileToUpload(getApkPath(), FileType.BASE));
     List<FileToUpload> splits = getSplits();
-    for (FileToUpload split : splits) {
-      if (split.getPath()
-          .contains("split_config")) {
-        list.add(split);
-      }
-    }
+    list.addAll(splits);
```

Comment:
Just a question: Won't this also try to upload the base apk, since it is in the same dir as the splits?

Reply:
I logged these files and it includes only the splits, not the apk